taqueria-vscode-extension-web-ui/src/data-editors/OptionEditor.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { DataEditorNode } from './DataEditorNode';
+import { VSCodeCheckbox } from './VsCodeWebViewUIToolkitWrappers';
+
+export const OptionEditor = (
+	{ dataType, value, onChange }: { dataType: any; value: any; onChange: (value: any) => void },
+) => {
+	const changeValue = (v: any) => {
+		onChange({
+			'prim': 'Some',
+			'args': [v],
+		});
+	};
+	if (
+		value === null || value === undefined || typeof value !== 'object'
```

Comment:
I think this expression could probably be simplified to make it easier to read. Perhaps:
```
!['None', 'Some'].includes(value.prim)
```